ovh/resource_ip_move_test.go

```diff
@@ -0,0 +1,48 @@
+package ovh
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
```

Comment:
```suggestion
	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
```

(or else we won't be able to run the tests anymore)